src/components/productList/Filter/index.js

```diff
@@ -5,19 +5,29 @@ import { useData } from "../../../Context/stateManage/state-context";
 import { CheckboxBrand } from "./CheckboxBrand";
 
 const Filter = () => {
-  const { dispatch } = useData();
+  const {
+    dispatch,
+    state: { selectedFilters },
+  } = useData();
 
```

Comment:
Suggestion : using the same name dispatch can be confusing. Using specific naming conventions can help.

Reply:
I will change it.